src/sentry/migrations/0517_backfill_pagerdutyservices_into_org_integrations.py

```diff
@@ -0,0 +1,37 @@
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapper
+
+
+def backfill_pagerdutyservices(apps, schema_editor):
+    PagerDutyService = apps.get_model("sentry", "PagerDutyService")
+
+    for pds in RangeQuerySetWrapper(PagerDutyService.objects.all()):
+        pds.save()
```

Comment:
Are you relying on a signal to fire here or something? That won't work in a migration

Reply:
No, I"m rely on the save method override, not a signal.  It, does preserve the model code + save methods, right?